src/policy/space.rs

```diff
@@ -42,6 +43,9 @@ use downcast_rs::Downcast;
 pub trait Space<VM: VMBinding>: 'static + SFT + Sync + Downcast {
     fn as_space(&self) -> &dyn Space<VM>;
     fn as_sft(&self) -> &(dyn SFT + Sync + 'static);
+    fn as_inspector(&self) -> Option<&dyn SpaceInspector> {
```

Comment:
We should just return `&dyn SpaceInspector`. It is temporary to use an `Option` so I don't have to implement for all the spaces.